_posts/2018-08-19-openfaas-on-fargate.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Faas on Fargate&#58; run OpenFaaS on AWS Fargate
+description: Edward Wilde introduces running OpenFaaS on AWS Fargate - a way to deploy and run functions without managing servers
+date: 2018-08-02
+image: /images/openfaas-on-fargate/astronomy-atmosphere-earth-220201.jpg
+categories:
+  - fargate
+author_staff_member: ed
+---
```

Comment:
How about start with an intro paragraph that sums up the whole post?

Reply:
added intro

`This article will explain why I've chosen to run OpenFaaS on AWS Fargate. We then move on to explore how to deploy a fully working environment and run our first function`